pkg/pillar/containerd/oci.go

```diff
@@ -535,3 +531,24 @@ func (s *ociSpec) UpdateEnvVar(envVars map[string]string) {
 		}
 	}
 }
+
+// UpdateShimVMDevices grants full access to file devices for shim VM OCI specs
+func (s *ociSpec) UpdateShimVMDevices() {
+	if len(s.Get().Process.Args) == 0 {
+		return
+	}
+
+	ctrCmd := s.Get().Process.Args[0]
```

Comment:
Is this derived from the `ENTRYPOINT` of an image?

I am asking, because I fear somebody could create an image like this:
```
FROM alpine:latest
RUN cp /bin/sh /bin/qemu-system-sh

ENTRYPOINT /bin/qemu-system-sh
```

Reply:
Yes, I've converted the PR to draft, let me check a bit more deep.... maybe we don't need to enforce the device access on Load() if the other functions are always executed.... in this case it becomes much more simpler and I can get rid of this checking....

Reply:
I think `AddLoader` is a good place to call `UpdateShimVMDevices` because that's where we add the Args and it's always called last during the container creation.